packages/@aws-cdk/core/test/test.staging.ts

```diff
@@ -273,6 +342,7 @@ export = {
     // THEN
     test.throws(() => new AssetStaging(stack, 'Asset', {
       sourcePath: directory,
+      assetHashType: AssetHashType.BUNDLE,
```

Comment:
Ah! Nice catch. I can undo this change. While I was working, this test was failing because the bundle asset already existed in the .cdk.staging from the previous tests, so docker wasn't being run at all so that it could throw - putting it to BUNDLE asset hash type, made it to run every time because it reverted the behaviour to randomly named directories. Now that .cdk.staging is cleaned on test teardown, it's a non-issue. I'll put it back. :)